gnosis/dx-react
# DX-671 // TOS vs Network Check

# DX-671 // TOS vs Network Check

DX-671

Checks current network to shape disclaimer accepted logic. First time access to networks requires verification approval.

To Test: 
1. clear `settings` in localForage. 
2. Use network `Rinkeby` and go through verification (check it goes to wallet selector after failing to find `TokenMGNProxy` (will be fixed later)
3. Check `localForage` to see that `settings` has `disclaimer_accepted: true` `&&` `networks_accepted: { RINKEBY: true }`
4. Switch to `Main` - make sure it opens up verification again.
5. Verify logic works by switching to `Rinkeby` and ensuring you do NOT have to re-check verification. It should go straight to wallet failing place
